jenkins/README.md

```diff
@@ -3,7 +3,7 @@
 Some integration tests are running in the [Nordix](https://www.nordix.org)
 infrastructure. Nordix provides a
 [Jenkins](https://jenkins.nordix.org/view/Airship/) instance and cloud resources
-on [CityCloud](https://www.citycloud.com/) for the Airship project. We use those
+on [CityCloud](https://citycontrolpanel.com/) for the Airship project. We use those
```

Comment:
I think this is not needed, because I would assume that apart from EST, others don't have an account in citycloud.

Reply:
Done